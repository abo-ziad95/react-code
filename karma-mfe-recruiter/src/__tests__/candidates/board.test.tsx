import React from 'react';
import renderer from 'react-test-renderer';
import Karma from "@hatech/karma-core";
import Board from "../../views/candidates/board";

it("Board component renders correctly", () => {
    const board = renderer
        .create(<Karma><Board/></Karma>)
        .toJSON();
    expect(board).toMatchSnapshot();
});