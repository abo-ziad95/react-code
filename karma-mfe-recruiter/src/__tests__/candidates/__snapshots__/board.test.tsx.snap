// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Board component renders correctly 1`] = `
<div>
  <div
    className="wrapper"
  >
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 MuiGrid-justify-xs-center"
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-3 MuiGrid-grid-lg-3 MuiGrid-grid-xl-3"
      >
        <h6
          className="MuiTypography-root MuiTypography-h6"
        >
          Applied
        </h6>
        <hr
          className="MuiDivider-root"
          style={
            Object {
              "margin": "12px 0px",
            }
          }
        />
        <div
          style={
            Object {
              "background": "none",
              "height": "100%",
              "width": "100%",
            }
          }
        />
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-3 MuiGrid-grid-lg-3 MuiGrid-grid-xl-3"
      >
        <h6
          className="MuiTypography-root MuiTypography-h6"
        >
          Pre-Screened
        </h6>
        <hr
          className="MuiDivider-root"
          style={
            Object {
              "margin": "12px 0px",
            }
          }
        />
        <div
          style={
            Object {
              "background": "none",
              "height": "100%",
              "width": "100%",
            }
          }
        />
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-3 MuiGrid-grid-lg-3 MuiGrid-grid-xl-3"
      >
        <h6
          className="MuiTypography-root MuiTypography-h6"
        >
          Interview
        </h6>
        <hr
          className="MuiDivider-root"
          style={
            Object {
              "margin": "12px 0px",
            }
          }
        />
        <div
          style={
            Object {
              "background": "none",
              "height": "100%",
              "width": "100%",
            }
          }
        />
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-3 MuiGrid-grid-lg-3 MuiGrid-grid-xl-3"
      >
        <h6
          className="MuiTypography-root MuiTypography-h6"
        >
          Hired
        </h6>
        <hr
          className="MuiDivider-root"
          style={
            Object {
              "margin": "12px 0px",
            }
          }
        />
        <div
          style={
            Object {
              "background": "none",
              "height": "100%",
              "width": "100%",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;
